tools/openblas_support.py

```diff
@@ -43,27 +43,25 @@ def download_openblas(target, arch, ilp64):
     fnsuffix = {None: "", "64_": "64_"}[ilp64]
     filename = ''
     if arch in ('aarch64', 'ppc64le', 's390x'):
-        filename = '{0}/openblas{1}-{2}-manylinux2014_{3}.tar.gz'.format(
-                        RACKSPACE, fnsuffix, OPENBLAS_LONG, arch)
+        filename = f'{RACKSPACE}/openblas{fnsuffix}-{OPENBLAS_LONG}-manylinux2014_{arch}
         typ = 'tar.gz'
         typ = 'tar.gz'
     elif arch == 'darwin':
-        filename = '{0}/openblas{1}-{2}-macosx_10_9_x86_64-gf_1becaaa.tar.gz'.format(
-                        RACKSPACE, fnsuffix, OPENBLAS_LONG)
+        filename = f'{RACKSPACE}/openblas{fnsuffix}-{OPENBLAS_LONG}-macosx_10_9_x86_64-gf_1becaaa.tar.gz'
```

Comment:
Line looks too long. We try to keep lines < 80 chars.  A separate variable for `macosx_10_9_x86_64-gf_1becaaa.tar.gz` is probably a good way to go. Maybe`MAC_VERSION` or some such.

Reply:
@tylerjereddy @mattip Is the library current? Do we even need this any more now that there is `tools/openblas_support.py`

Reply:
remember you can split f-strings over multiple lines as `f'{a}' f'{b}'`

Reply:
> Line looks too long. We try to keep lines < 80 chars. A separate variable for `macosx_10_9_x86_64-gf_1becaaa.tar.gz` is probably a good way to go. Maybe`MAC_VERSION` or some such.
Should I make the change?

Reply:
Of course, please do.